.buildkite/commands/configure-environment.sh

```diff
@@ -1,7 +0,0 @@
-#!/bin/bash -eu
-
-echo '--- :git: Configure Git for Release Management'
-.buildkite/commands/configure-git-for-release-management.sh
-
-echo '--- :ruby: Set up Ruby Tools'
-install_gems
```

Comment:
This script could have come in handy in the future, to be honest. But in looking at it I realized a minor issue: it installs the gems before switching branch. This is unlikely to cause issues, but it might in some rare occasions. Given this limitation and the fact that it's used only once at the moment (YAGNI) I decided to remove it and delegate my future self to DRY in a better way if required.